tests/network-tests/src/fixtures/workingGroups/SpendBudgetFixture.ts

```diff
@@ -9,15 +9,27 @@ import { ISubmittableResult } from '@polkadot/types/types/'
 import { Utils } from '../../utils'
 import { BudgetSpendingEventFieldsFragment, WorkingGroupFieldsFragment } from '../../graphql/generated/queries'
 
+export type VestingSchedule = {
+  locked: BN
+  perBlock: BN
+  startingBlock?: BN
+}
+
 export class SpendBudgetFixture extends BaseWorkingGroupFixture {
   protected recievers: string[]
-  protected amounts: BN[]
+  protected vestingSchedules: VestingSchedule[]
```

Comment:
Should this be changed ? The fixture is still called `SpendBudgetFixture` not `SpendVestedBudgetFixture`

Reply:
change reverted, integration tests needs to be taken care of in a separate PR. 
Namely add this Fixture with Flow